tutorials/plot_receptive_field.py

```diff
@@ -0,0 +1,206 @@
+"""
+=============================================
+Receptive field estimation on continuous data
+=============================================
+
+This demonstrates how an encoding model can be fit with multiple continuous
+inputs. In this case, we simulate the model behind a spectro-temporal receptive
+field (or STRF). First, we create a linear filter that maps patterns in
+spectro-temporal space onto an output, representing neural activity. We fit
+a receptive field model that attempts to recover the original linear filter
+that was used to create this data.
+
+Estimation of spectro-temporal and spatio-temporal receptive fields using
+modeling with continuous inputs is described in:
+
+    [1] Theunissen, F. E. et al. Estimating spatio-temporal receptive
```

Comment:
please fix these too, like `as described in [1]_ [2]_ [3]_` and add References section the standard way